.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: QA DEPLOY
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: authenticate npm
+      run: npm set //npm.pkg.github.com/:_authToken ${{ secrets.PACKAGING_TOKEN }}
+
+    - run: git config --global user.name kaiomagalhaes
```

Comment:
We don't need to set personal information to publish packages.
There are more info on this here [https://docs.github.com/en/free-pro-team@latest/actions/guides/publishing-nodejs-packages#publishing-packages-to-github-packages](https://docs.github.com/en/free-pro-team@latest/actions/guides/publishing-nodejs-packages#publishing-packages-to-github-packages)